users/models.py

```diff
@@ -39,8 +39,6 @@ def get_with_prefetched_reports(self, reports: QuerySet) -> QuerySet:
 
 
 class CustomUserManager(BaseUserManager):
-    use_in_migrations = True
```

Comment:
The usage of this property is obscure. According to [Django documentation](https://docs.djangoproject.com/en/2.2/topics/migrations/#model-managers) it has to do with serialisation. @Szymiks reported seeing this on forums regarding REST framework. This line was created by @MartynaAnnaGottschling back in October 2018. `full-check` did not raise any errors after it got removed.

Is it therefore safe to assume that this is a leftover from the REST implementation and can be safely deleted?

Reply:
Not related with REST exactly, but with possibility to use custom manager methods in migrations. As we don't use them, lets remove it.